### PR TITLE
Delete ~180 lines of footer CSS from adjustments.css

### PR DESCRIPTION
**Pull request by nfagerlund**
The footer CSS is some of our worst remaining jank, and this commit removes about 40% of it, to basically no observable effect. (The spacing differs by about 5-10 px in various places, but you only notice it if you flip back and forth between tabs in the same browser window.) 

At least, I think there's no observable effect; I could use another pair of eyes to make sure I'm not missing anything significant. @gguillotte, could you review this?

### CODE REVIEW

File: source/files/stylesheets/adjustments.css [hunk 1]
```diff
@@ -317,139 +311,11 @@ ul.doc-navigation {
   transition: color 0.2s linear;
 }
 
-.layout-footer .footer-row-1 {
-  margin: 0 auto;
-/*
-  max-width: 67em;
- */
-  padding: 0 10px;
-  min-height: 260px;
```
Comment (gguillotte) [inline source/files/stylesheets/adjustments.css]: Without this min-height, we lose the negative space between the bottom of the footer content and the trailing line of legal/copyright links.

File: source/files/stylesheets/adjustments.css [hunk 2]
```diff
@@ -317,139 +311,11 @@ ul.doc-navigation {
...
-}
-
-.layout-footer .footer-row-1:before,
-.layout-footer .footer-row-1:after {
-  content: "";
-/*
```
Comment (gguillotte) [inline source/files/stylesheets/adjustments.css]: This is an IE hack; without it, the footer will bleed into the content on IE 11.

File: source/files/stylesheets/adjustments.css [hunk 3]
```diff
@@ -317,139 +311,11 @@ ul.doc-navigation {
...
-.layout-footer .footer-row-1 > .fg-col:first-child {
-  box-sizing: border-box;
-}
- */
-
-.layout-footer .footer-row-1 > .fg-col:first-child:before,
-.layout-footer .footer-row-1 > .fg-col:first-child:after {
-  content: " ";
-/*
```
Comment (gguillotte) [inline source/files/stylesheets/adjustments.css]: This is an IE hack; without it, the footer will bleed into the content on IE 11.

File: source/files/stylesheets/adjustments.css [hunk 4]
```diff
@@ -317,139 +311,11 @@ ul.doc-navigation {
...
-
-/*
-.layout-footer .footer-row-1 > .fg-col:last-child {
-  box-sizing: border-box;
-}
- */
-
-.layout-footer .footer-row-1 > .fg-col:last-child:before,
-.layout-footer .footer-row-1 > .fg-col:last-child:after {
-  content: " ";
```
Comment (gguillotte) [inline source/files/stylesheets/adjustments.css]: IE hack.

File: source/files/stylesheets/adjustments.css [hunk 5]
```diff
@@ -465,20 +331,10 @@ html:not(.csscalc) .layout-footer .footer-row-1 > .fg-col:last-child {
   margin-top: 10px;
 }
 
-.layout-footer .footer-row-1 > .fg-col p {
-  margin: 0 0 20px 0;
```
Comment (gguillotte) [inline source/files/stylesheets/adjustments.css]: This produces the space between the mailing list copy and form field.

File: source/files/stylesheets/adjustments.css [hunk 6]
```diff
@@ -487,18 +343,12 @@ html:not(.csscalc) .layout-footer .footer-row-1 > .fg-col:last-child {
   height: auto;
 }
 
-.layout-footer .footer-row-1 .social-icon {
-  float: left;
```
Comment (gguillotte) [inline source/files/stylesheets/adjustments.css]: Removing this fixes the social media icons on IE 11, but changes their spacing on other browsers.

File: source/files/stylesheets/adjustments.css [hunk 7]
```diff
@@ -487,18 +343,12 @@ html:not(.csscalc) .layout-footer .footer-row-1 > .fg-col:last-child {
...
-  margin-top: 10px;
```
Comment (gguillotte) [inline source/files/stylesheets/adjustments.css]: Removing this causes the social media icons to directly abut the input field.

File: source/files/stylesheets/adjustments.css [hunk 8]
```diff
@@ -487,18 +343,12 @@ html:not(.csscalc) .layout-footer .footer-row-1 > .fg-col:last-child {
...
-}
-
-.layout-footer .footer-row-1 .social-icon + .social-icon {
-  margin-left: 10px;
```
Comment (gguillotte) [inline source/files/stylesheets/adjustments.css]: Removing this removes the spacing between social media icons.

**Comment by gguillotte** [comment]
@nfagerlund There are a few :before and :after rules that, if cut, will break page rendering on IE 11.

File: source/files/stylesheets/adjustments.css (hunk 6 above)
Comment (nfagerlund) [inline source/files/stylesheets/adjustments.css]: Hmm. Do you think the changed spacing looks bad, or should we just go for it? I can't actually discern the difference on Gecko or Blink, tbh.

**Comment by nfagerlund** [comment]
OK, I restored the IE11 hacks, the min-height, and the mailing list spacing. I can't see any spacing differences in the social media icons, but you're saying they're not squashed on IE now?

**Comment by gguillotte** [comment]
@nfagerlund LGTM.

**Comment by nfagerlund** [comment]
sweet

**Pull request merged**